src/pages/en/guides/imports.md

```diff
@@ -210,3 +210,40 @@ const data = JSON.parse(json);
 
 <span>Version: {data.version}</span>
 ```
+
+## Extending file type support
+
+With **Vite** and compatible **Rollup** plugins, you can import file types which aren't natively supported by Astro. Learn where to find the plugins you need in the [Finding Plugins](https://vitejs.dev/guide/using-plugins.html#finding-plugins) section of Vite Documentation.
+
+### Example: YAML support
```

Comment:
```suggestion

 :::note[plugin configuration]
Refer to your plugin's documentation for configuration options, and how to correctly install it.
:::

### Example: YAML support
```

Reply:
Does this make it more clear that config/installing might look different for each plugin, and then below is one specific example?

(If so, then we can remove the note from lower in the page.)

Reply:
Sure! Does it even need to be a note?

Reply:
Probably not!

Reply:
I'm going to commit it as a note first, to see how it looks. The advantage of a note is making sure it's clear that not EVERY plugin is as straightforward as the example given below?

I'll see if it looks too overwhelming.